indico/modules/events/contributions/controllers/management.py

```diff
@@ -28,6 +28,7 @@
 from indico.modules.events.abstracts.forms import AbstractContentSettingsForm
 from indico.modules.events.abstracts.settings import abstracts_settings
 from indico.modules.events.contributions import get_contrib_field_types
+from indico.modules.events.contributions.controllers.base import ContributionListMixin
```

Comment:
`s/base/common/`? I'd rather use `base.py` for classes that are either bases or generic (not specific to one feature) mixins